libraries/AdaptiveExpressions/parser/ExpressionParser.cs

```diff
@@ -164,6 +164,11 @@ public override Expression VisitNumericAtom([NotNull] ExpressionAntlrParser.Nume
                     return Expression.ConstantExpression(intValue);
                 }
```

Comment:
Why not drop this and always just do long?

Reply:
Converting all to long will cause some errors, because some users' codes are written with "int32"